### PR TITLE
build(dependabot): Add dockerfiles to dependabot config

### PR DESCRIPTION
**Pull request by hairyhum**
## Change Overview

Add docker images to dependabot config to keep image dependencies up to date.

## Test Plan

Need to monitor dependabot PRs and vulnerability scanner reports to see if that has any positive effect

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -49,3 +51,130 @@ updates:
       docker:
         patterns:
         - "docker/*"
+  ## Currently dependabot does not support wildcard or multiple directories
```
Comment (julio-lopez) [inline .github/dependabot.yml:54]: @hairyhum Does it support multiple stanzas / entries for the same package ecosystem? that is multiple `package-ecosystem: "docker"` ?

**Review by julio-lopez — APPROVED** [review]
Feel free to merge and we can test / tweak as needed.

File: .github/dependabot.yml (hunk 1 above)
Comment (hairyhum) [reply .github/dependabot.yml:54]: It's not officially documented, but people use it like that.

Comment (hairyhum) [reply .github/dependabot.yml:54]: https://github.com/dependabot/dependabot-core/issues/2178

**Pull request merged**